UI/src/app/models/arrow.ts

```diff
@@ -119,13 +120,15 @@ export class Arrow implements IConnector {
     this.button = button;
   }
 
-  setEndMarkerType(type: string): void {
+  setEndMarkerType(type): void {
     this.refreshPathHtmlElement();
 
     const isActive = this.svgPath.attributes[markerEndAttributeIndex].value.includes('active');
 
     this.renderer.removeAttribute(this.svgPath, 'marker-end');
-    this.renderer.setAttribute(this.svgPath, 'marker-end', `url(#marker-end${isActive ? '-active' : ''}${type !== 'None' ? `-${type}` : ''})`);
+    type = type === 'None' ? '' : type;
+    this.renderer.setAttribute(this.svgPath, 'marker-end', `url(#marker-end${isActive ? '-active' : ''}${type ? `-${type}` : ''})`);
```

Comment:
Is this `url(#marker-end${isActive ? '-active' : ''}${type ? ` unclosed type variable?

Reply:
Shouldn't it be 
```ts
this.renderer.setAttribute(this.svgPath, 'marker-end', `url(#marker-end${isActive ? '-active' : ''}${type} ? `-${type}` : ''})`);
```
?

Probably it's better to split the logic of checking these properties.
```ts
const markerActive= isActive ? '-active' : '';
const markerType= type ? `-${type}` : '';
this.renderer.setAttribute(this.svgPath, 'marker-end', `url(#marker-end${markerActive}${markerType})`);
```

Reply:
refactored to make logic clear